### PR TITLE
Change some arg-names to match Python

### PR DESCRIPTION
**Pull request by johnkerl**
Change from `display_name` to `name`, and `do_local` to `local`.

See for reference https://docs.tiledb.com/cloud/api-reference/task-graphs.

### CODE REVIEW

File: R/manual_layer_node.R [hunk 1]
```diff
@@ -33,7 +33,7 @@ Node <- R6::R6Class(
     # insist that people separately articulate that. See the 'delayed' factory
     # function.
     have_args     = NULL,
-    do_local      = NULL,
+    local      = NULL,
```
Comment (thetorpedodog) [inline R/manual_layer_node.R:36]: nit: alignment

File: R/manual_layer_node.R [hunk 2]
```diff
@@ -44,11 +44,11 @@ Node <- R6::R6Class(
     dag_for_terminal = NULL,
 
     id            = NULL,
-    display_name  = NULL,
+    name  = NULL,
```
Comment (thetorpedodog) [inline R/manual_layer_node.R:47]: here too

File: R/manual_layer_node.R [hunk 3]
```diff
@@ -58,12 +58,12 @@ Node <- R6::R6Class(
...
-      if (is.null(display_name)) {
-        self$display_name <- self$id
+      if (is.null(name)) {
+        self$name <- self$id
       } else {
-        self$display_name <- display_name
+        self$name <- name
       }
-      self$do_local     <- do_local
+      self$local     <- local
```
Comment (thetorpedodog) [inline R/manual_layer_node.R:66]: Did this used to be grouped with the stuff around L58?

**Review by thetorpedodog — APPROVED** [review]

**Comment by johnkerl** [comment]
thanks @thetorpedodog !!

**Pull request merged**